theano/gpuarray/opt.py

```diff
@@ -82,6 +105,28 @@ def f(local_opt):
     return f
 
 
+def register_opt2(tracks, *tags, **kwargs):
+    '''
+    Decorator for the new GraphToGPU optimizer.
+    Takes an extra parameter(Op) compared to register_opt decorator.
+
+    Parameters
+    ----------
+    tracks : Op
```

Comment:
This is a list of Op class or Op instance or None (to be applied on all node in the graph)